src/utils/num.ts

```diff
@@ -187,6 +317,13 @@ export function addPercent(number: BigNumberish, percent: number) {
  *
  * @param {unknown} value - The value to check.
  * @return {boolean} Returns true if the value is a number, otherwise returns false.
+ * @example
+ * ```typescript
+ * const result = isNumber(123);
+ * // result = true
+ * 
+ * const result2 = isNumber("123");
+ * // result = true
```

Comment:
Are you sure? Have you tested it?

Reply:
Yes i did, 
i made a typo error
corrected : "// result = true"  to "// result2 = false"